components/p3a/rotation_scheduler_unittest.cc

```diff
@@ -58,34 +53,29 @@ TEST_F(P3ARotationSchedulerTest, JsonRotation) {
   EXPECT_EQ(json_rotation_counts_[MetricLogType::kSlow], 3u);
 }
 
-// TODO(djandries): find a way to get this test to run reliably on macOS
-#if BUILDFLAG(IS_MAC)
-#define MAYBE_ConstellationRotation DISABLED_ConstellationRotation
-#else
-#define MAYBE_ConstellationRotation ConstellationRotation
-#endif
-
-TEST_F(P3ARotationSchedulerTest, MAYBE_ConstellationRotation) {
+TEST_F(P3ARotationSchedulerTest, ConstellationRotation) {
   task_environment_.FastForwardBy(base::Days(7));
   // Should be 0 since the timer has not started
   EXPECT_EQ(constellation_rotation_count_, 0u);
 
-  scheduler_->InitConstellationTimer(task_environment_.GetMockClock()->Now() +
```

Comment:
this is the main issue. `task_environment_.GetMockClock()` returns different values than `base::Time::Now()`. Looks like an upstream bug.

Reply:
good catch, thanks. had trouble figuring this out since i don't have macos